### DISCUSSION TITLE
cn/2024/02/how-many-programming-languages-do-we-need/

### DISCUSSION THREAD

**Discussion by giscus** [leovan/leovan.me#39] [Comments]
# cn/2024/02/how-many-programming-languages-do-we-need/

编程语言「只是」达成目标的工具，这是我一直推崇的说法，因为我认为达成目标更重要的在于个人思考，编程语言不过是个「工具」，选哪个并没有那么重要

https://leovan.me/cn/2024/02/how-many-programming-languages-do-we-need/?utm_source=blogfinder

**Comment by 7wate** [comment]
赞同加一哈哈，人生苦短，我用 Python。
 目前计划学习了解一下 Rust！

**Comment by AbstractWorld** [comment]
我要用Go😄

**Comment by XiangyunHuang** [comment]
深表赞同👍 之前的学习和工作是用R，我去年开始也兼学一点 Python 。
